Profiles - ZIB 2017/zib-LaboratoryTestResult-DiagnosticReport.xml

```diff
@@ -61,7 +63,8 @@
     <element id="DiagnosticReport">
       <path value="DiagnosticReport" />
       <short value="LaboratoryTestResult" />
-      <definition value="The findings and interpretation of diagnostic tests performed on patients, groups of patients, devices, and locations, and/or specimens derived from these. The report includes clinical context such as requesting and provider information, and some mix of atomic results, images, textual and coded interpretations, and formatted representation of diagnostic reports. &#xD;&#xA;The HCIM LaboratoryResult also supports simple and more complex situations. For a simple result there will be a single LaboratoryTestResult.LaboratoryTest.TestName in Observation.code and no LaboratoryTestresult.Test. For a more complex situation there will be a LaboratoryTestResult.Test and any number of related LaboratoryTestResult.LaboratoryTest, e.g. parts of a panel or battery. Each related LaboratoryTest shall be conveyed in Observation.related or Observation.component. The deciding factor in choosing between related or component is: does the observation have use as independent observation (Observation.related) or is the observation only relevant in the context of its parent (Observation.component).&#xD;&#xA;Note that (textual) reports on individual (panels of) observations, like conclusions, go into a different resource DiagnosticReport. The DiagnosticReport then contains the references to the Observation resources it concerns." />
+      <definition value="A laboratory result describes the result of a laboratory analysis.&#xD;&#xA;These are specimen-oriented tests as performed in laboratories such as Clinical Chemistry, Serology, Microbiology, etc.&#xD;&#xA;In addition to the results of tests with a singular result, this concept can also contain the results of more complex tests with multiple results or a ‘panel’." />
+      <comment value="This profile is intended to capture a single report, and is not suitable for use in displaying summary information that covers multiple reports.  For example, this resource has not been designed for laboratory cumulative reporting formats nor detailed structured reports for sequencing.&#xD;&#xA;&#xD;&#xA;The findings and interpretation of diagnostic tests performed on patients, groups of patients, devices, and locations, and/or specimens derived from these. The report includes clinical context such as requesting and provider information, and some mix of atomic results, images, textual and coded interpretations, and formatted representation of diagnostic reports.&#xD;&#xA;&#xD;&#xA;The HCIM LaboratoryTestResult supports simple and more complex situations. This profile is used for (textual) reports on individual (panels of) observations, like conclusions. The actual tests are captured using the profile [zib-LaboratoryTestResult-Observation](http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult-Observation), which should be referenced from this resource." />

```

Comment:
Perhaps we want to put more emphasis on our added text to this comment section. We could start with "The HCIM LaboratoryTestResult supports simple and more complex....." and then the standard FHIR or perhaps, even better, remove the FHIR spec wording?

Reply:
Ok